apps/ledger-live-mobile/src/screens/Settings/General/AuthSecurityToggle.tsx

```diff
@@ -7,17 +7,27 @@ import { NavigatorName } from "../../../const";
 import { privacySelector } from "../../../reducers/settings";
 import SettingsRow from "../../../components/SettingsRow";
 import BiometricsRow from "./BiometricsRow";
+import { ScreenName } from "../../../const";
+import { track } from "../../../analytics";
 
 export default function AuthSecurityToggle() {
   const { t } = useTranslation();
 
   const privacy = useSelector(privacySelector);
   const { navigate } = useNavigation();
 
-  const onValueChange = (authSecurityEnabled: boolean): void =>
+  const onValueChange = (authSecurityEnabled: boolean): void => {
+    // Add onClick Analytics
+    track("toggle_clicked", {
+      toggle: "Password Lock", // Moyen de chopper la key non traduite ?
```

Comment:
No, AFAIK stuff like that are in English. You can also remove the `// Add onClick Analytics` it's already pretty straightforward IMO.